src/myFunc.cpp

```diff
@@ -1552,11 +1552,29 @@ float runEM (const vector<float>& x,const vector<float>& y,double * a, int degre
 
 	for (count = 0; count < NumberOfIterations; count++) {
 
+		// assign best ploidy for each window 
+		std::vector<float> res_at_ploidy;
```

Comment:
Are we still using this change?

Reply:
yes, this is used for  the case of choosing with running with polynomial fit. 
I think this is a reasonable contribution.